pages/apply.html

```diff
@@ -42,18 +42,31 @@
 
         <div class="usa-input">
           <input type="checkbox" name="entry.1377828897" value="Verify that you meet the above eligibility requirements" id="group_1377828897_1" role="checkbox" class="ss-q-checkbox">
-          <label for="group_1377828897_1">Verify that you meet the above eligibility requirements</label>
+          <label for="group_1377828897_1">I, the applicant, verify that I meet the above eligibility requirements.</label>
+          <label for="group_1377828897_2">I, the applicant, verify that I do not meet the above eligibility requirements and will not be completing the rest of this application.</label>
```

Comment:
@hestherchang Please use the Google form admin interface to make this a radio button. (Rationale is explained below.) Thanks!